libs/server/Storage/Functions/ObjectStore/ReadMethods.cs

```diff
@@ -37,6 +37,15 @@ public bool SingleReader(ref byte[] key, ref ObjectInput input, ref IGarnetObjec
                 if ((byte)input.header.type < CustomCommandManager.StartOffset)
                     return value.Operate(ref input, ref dst.spanByteAndMemory, out _, out _);
 
+                if (input.header.type == GarnetObjectType.Expiretime || input.header.type == GarnetObjectType.PExpiretime)
```

Comment:
Can create a single switch with all variants of TTL and ExpireTime, to consolidate.

Reply:
Added `switch` for the entire type so that in future if there is more `GarnetObjectType` that needs to be handled in the ObjectSession layer itself, we can use the same switch